CHANNGELOG.md

```diff
@@ -2,7 +2,8 @@ authconfig Cookbook Changelog
 =============================
 This file is used to list changes made in each version of the authconfig cookbook.
 
-v1.1.13
--------
-### BUG
-- Avoid sssd start or restart if sssd daemon disabled
+## 2.0.5
+* Enable/start sssd if it is being installed
```

Comment:
Was originally thinking of conditions to get to https://github.com/mitch-roblox/authconfig/blob/6cb41ab88f91806e31613cda2767b7e9059a3f4b/recipes/default.rb#L165 if the config changes and if it reloads? (I acknowledge it might be outside scope of this PR, just not sure if it would impact how `sssd_service_action` is used)